hack/e2e-test.sh

```diff
@@ -104,6 +113,12 @@ function teardown() {
 }
 trap 'teardown' EXIT
 
+function save_hive_logs() {
+  oc logs -n hive deployment/hive-operator &> "${ARTIFACT_DIR}/hive-operator.log"
```

Comment:
We don't need to capture this. The hive-operator will still be running after the HiveConfig CR is deleted. The normal e2e capture will already capture this.